### DISCUSSION TITLE
Few questions on configurability of DHCP server

### DISCUSSION THREAD

**Discussion by chrislawso** [BeryJu/gravity#1166] [General]
Is the DHCP server able to perform ip reservations and serve preconfigured MAC addresses reserved/preset ip addresses?

Can the DHCP and/or DNS be configured such that if machines specify their hostname when requesting an IPv4 DHCP lease is the DNS able to register the hostname so that their name can be resolved?

Thank you

**Comment by tompaah** [comment]
> Is the DHCP server able to perform ip reservations and serve preconfigured MAC addresses reserved/preset ip addresses?

Yes it is able to do that..

> Can the DHCP and/or DNS be configured such that if machines specify their hostname when requesting an IPv4 DHCP lease is the DNS able to register the hostname so that their name can be resolved?

Yes, it appears it works that way.
